go.mod

```diff
@@ -2,8 +2,8 @@ module github.com/terraform-providers/terraform-provider-grafana
 
 require (
 	github.com/blang/semver v3.5.1+incompatible // indirect
+	github.com/emerald-squad/go-grafana-api v0.0.0-20190510131408-a95b9575e28e
```

Comment:
Could you get a PR of the required changes to support orgs to the upstream library?